sapi/fpm/fpm/fpm_log.c

```diff
@@ -205,8 +205,8 @@ int fpm_log_write(char *log_format) /* {{{ */
 							len2 = snprintf(b, FPM_LOG_BUFFER - len, "%.3f", proc.duration.tv_sec + proc.duration.tv_usec / 1000000.);
 						}
 
-					/* miliseconds */
-					} else if (!strcasecmp(format, "miliseconds") || !strcasecmp(format, "mili")) {
+					/* milliseconds */
+					} else if (!strcasecmp(format, "milliseconds") || !strcasecmp(format, "milli")) {
```

Comment:
I'm not sure if it is a break, as it might only be called internally so if all other occurrences are fixed should be fine.

Reply:
This is a break. It even breaks the example in the config template: https://gist.github.com/sotarok/2767859#file-php-fpm-conf-L406 (if someone decides to uncomment it) so you need keep the incorrect one. I'm fine if you add a correct one as well and updated that commented line to use the correct one but please keep the old one as this could be a hard to catch break. You can also add comment that the incorrect one is kept just for compatibility reason.

Reply:
Amusingly, the [previous major spelling round](https://bugs.php.net/bug.php?id=78880) already made a mess here: 
specifically:
https://github.com/php/php-src/blob/0b4778c377a5753a0deb9cfc697d4f62acf93a29/sapi/fpm/www.conf.in#L272